apis/server/container_bridge.go

```diff
@@ -270,10 +270,6 @@ func (s *Server) updateContainer(ctx context.Context, rw http.ResponseWriter, re
 	if err := json.NewDecoder(req.Body).Decode(config); err != nil {
 		return httputils.NewHTTPError(err, http.StatusBadRequest)
 	}
-	// validate request body
-	if err := config.Validate(strfmt.NewFormats()); err != nil {
```

Comment:
I removed the request body validation in update, then I think it will pass the CI. @Ace-Tang @HusterWan